src/main/java/io/jlu/jerbot/commands/HelpCommand.java

```diff
@@ -0,0 +1,39 @@
+package io.jlu.jerbot.commands;
+
+import net.dv8tion.jda.core.events.message.MessageReceivedEvent;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class HelpCommand implements Command {
+
+    Map<String, String> commandDefinitions;
+
+    public HelpCommand() {
+        this.commandDefinitions = defineCommands();
```

Comment:
Here I'm declaring a separate map in defineCommands(), and then setting our class field to equal that map (In this case is it just a shift in variable reference and therefore won't be costly at all?). Wasn't sure if it is better to define the class field as a new map, and then **directly** .put() information into it.